app/lib/forms/aso_new_headteacher.rb

```diff
@@ -43,7 +43,7 @@ def options
   private
 
     def course
-      Course.find_by(id: wizard.store["course_id"])
+      wizard.query_store.course
```

Comment:
As the first step to changing how we locate courses I replaced all manual lookups for courses using the course_id into the query_store. That way, once the user is selecting course identifiers (e.g. "npq-executive-leadership") instead of course IDs (e.g. 3) we only need to change one lookup location.

Reply:
This is much nicer.